frontend/src/container/LogDetailedView/InfraMetrics/NodeMetrics.tsx

```diff
@@ -0,0 +1,140 @@
+import { Card, Col, Row, Skeleton, Typography } from 'antd';
```

Comment:
nit: The NodeMetrics and PodMetrics components seem pretty similar, should we have a reusable component that can render both the components?

Reply:
@ahmadshaheer Initially I kept this as a common component but in node metrics we will change query payloads based on the presence of attribute K8s.node.name and thus created separate components.

Reply:
@rahulkeswani101, are we going to build a reusable component for NodeMetrics and PodMetrics in a future PR?

Reply:
Yes, @ahmadshaheer. I  have added a comment as a todo for this change in the InfraMetrics component.